config/routes.rb

```diff
@@ -323,6 +323,8 @@
     get '/sign_up/cancel/' => 'sign_up/cancellations#new', as: :sign_up_cancel
     delete '/sign_up/cancel' => 'sign_up/cancellations#destroy', as: :sign_up_destroy
 
+    get '/redirect/return_to_sp/account_verified_cta' => 'idv/account_verified_cta_visited#show', as: :account_verified_sign_in_redirect
```

Comment:
A/N: Using this route pattern, but not putting the actual controller in the `Redirect` module as the `AccountVerifiedCtaVisitedController` doesn't follow the `Redirect::RedirectController` pattern.